promitor-agent-scraper/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and ( .Values.metricSinks.prometheusScrapingEndpoint.serviceMonitor.enabled ) (.Values.metricSinks.prometheusScrapingEndpoint.enabled ) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+{{- if .Values.metricSinks.prometheusScrapingEndpoint.serviceMonitor.labels }}
+  labels:
+{{ toYaml .Values.metricSinks.prometheusScrapingEndpoint.serviceMonitor.labels | indent 4 }}
```

Comment:
Same here, can we at least add our default labels?
https://github.com/promitor/charts/blob/main/promitor-agent-scraper/templates/deployment.yaml#L7

Reply:
Same here 👍